ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/search-form-interactions/dropdown.search-form-interactions.hbs

```diff
@@ -0,0 +1,17 @@
+{{!--
```

Comment:
Register this file in `component.less`

Reply:
Less file was not needed because search-form.less is dictating look and feel. Files was removed